pages.de/common/code.md

```diff
@@ -0,0 +1,28 @@
+# code
+
+> Visual Studio Code.
+> Mehr informationen: <https://github.com/microsoft/vscode>.
+
+- VS Code öffnen:
+
+`code`
+
+- Aktuelles Verzeichnis in VS Code öffnen:
+
+`code .`
+
+- Eine Datei oder einen Ordner in VS Code öffnen:
+
+`code {{path/to/file_or_directory}}`
+
+- Eine Datei oder einen Ordner im aktuell geöffnetem VS Code Fenster öffnen:
+
+`code --reuse-window {{path/to/file_or_directory}}`
```

Comment:
Please translate the arguments for get this easier for non-English speakers. like `path/to/file_or_directory`, `file1`, `path/to/configuration_file.xml`, etc. **in all pages**

Reply:
Fixed it!